sklearn/utils/_test_common/instance_generator.py

```diff
@@ -542,6 +542,17 @@
         ]
     },
     MDS: {"check_dict_unchanged": dict(max_iter=5, n_components=1, n_init=2)},
+    MLPClassifier: {
```

Comment:
It is good enough for me. Thanks @lorentzenchr 

@ogrisel mentioned to me stochastic solver and the equivalence test might not be best friend. So SGD/Adam + early-stopping should be omitted.

Reply:
Exactly what I did in https://github.com/scikit-learn/scikit-learn/pull/30155/commits/432bcc08e29c458aacb4d0fc5c674b4aa1ebed4b.